misc/ipv4_subnet.c

```diff
@@ -0,0 +1,60 @@
+#include <assert.h>
+#include <stdio.h>
+/**
+ * @file
+ * @brief
+ * [Explanation](https://www.softwaretestinghelp.com/subnet-mask-and-network-classes/)
+ * @author [XenoxJeager] (https://github.com/XenoxJeager)
```

Comment:
Markdown links do not have spaces.
```suggestion
 * @author [XenoxJeager](https://github.com/XenoxJeager)
```